src/Pkg.jl

```diff
@@ -537,6 +537,17 @@ const precompile_script = """
     ] add Te\t\t$CTRL_C
     ] st
     $CTRL_C
+    # Create simple artifact, bind it, then use it:
+    foo_hash = Pkg.Artifacts.create_artifact(dir -> touch(joinpath(dir, "foo")))
+    Pkg.Artifacts.bind_artifact!("./Artifacts.toml", "foo", foo_hash)
+    # Also create multiple platform-specific ones because that's a codepath we need precompiled
+    Pkg.Artifacts.bind_artifact!("./Artifacts.toml", "foo_plat", foo_hash; platform=platform_key_abi())
```

Comment:
Did this every work?

```
julia> using Pkg.Artifacts

julia> platform_key_abi
ERROR: UndefVarError: platform_key_abi not defined
```

Reply:
It's in Pkg.BinaryPlatforms if I remember correctly

Reply:
Doesn't seem like it..?

Reply:
Perhaps it got lost in a rebase somewhere; I distinctly remember adding a `using Pkg.BinaryPlatforms` as well as testing this and having it work.